libflux/src/semantic/types.rs

```diff
@@ -1,100 +1,120 @@
-use crate::semantic::sub::{Subst, Substitutable};
+use crate::semantic::{
+    fresh::Fresher,
+    sub::{Substitutable, Substitution},
+};
 use std::{
     cmp,
-    collections::{BTreeMap, BTreeSet, HashMap, HashSet},
-    fmt, result,
+    collections::{BTreeMap, BTreeSet, HashMap},
+    fmt,
 };
 
-// PolyType represents a generic parametrized type.
-//
-// TODO:
```

Comment:
Is this TODO complete or somehow not relevant anymore?

Reply:
Not relevant. I added a `normalize` method to `PolyType` which allows polytypes to be equated using the `PartialEq` trait.